nirc_ehr/src/org/labkey/nirc_ehr/table/NIRC_EHRCustomizer.java

```diff
@@ -974,6 +974,20 @@ private void customizeAnimalTable(AbstractTableInfo ds)
             col.setLabel("Alias");
             ds.addColumn(col);
         }
+        if (ds.getColumn("lastProtocol") == null)
+        {
+            var col = getWrappedCol(us, ds, "lastProtocol", "demographicsLastProtocol", "Id", "Id");
+            col.setLabel("Assignment - Final Protocol");
+            col.setDescription("This calculates the final protocol assignment for the animal.  This is distinct from active protocol assignment because it will return a protocol assignment for dead animals");
```

Comment:
Is it meant to be only for dead animals? if so, demographicsLastProtocol.sql should filter on dead animals?

Reply:
It's for both alive and dead animals. Unlike active protocol assignment which is only for alive animals.